relayer/src/main.rs

```diff
@@ -369,9 +373,13 @@ async fn main() {
             let checkpoint_light_client_address =
                 hex_utils::decode_h256(&common.checkpoint_light_client_address)
                     .expect("Failed to parse address");
-            let ethereum_event_client_address =
+            let _ethereum_event_client_address =
```

Comment:
As I see `ethereum_event_client_address` can be removed from `RelayErc20ArgsCommon`